castep_outputs/parsers/phonon_dos_file_parser.py

```diff
@@ -57,14 +57,17 @@ def fix(qdat):
             logger("Found DOS block")
 
             dos = defaultdict(list)
+            # First chunk is " BEGIN DOS   Freq (cm-1)  g(f)", thus need the 5th on
+            species = map(str.lower, block[0].split()[5:])
```

Comment:
Probably you already thought about this, but it's weird enough to ask about:

Is it _really_ appropriate to downcase element names and user labels? Can that ever lead to ambiguity? (I _think_ we're OK on the ambiguity front because of the colon. I hope CASTEP doesn't allow multiple user labels distinguished by case. I hope.)

Reply:
That is a good point, I think it was leftover from when I was trying to parse all the data-sets in one block.
```suggestion
            species = block[0].split()[5:]
```